malos/driver.proto

```diff
@@ -233,73 +235,332 @@ message EverloopImage {
   repeated LedValue led = 1;
 }
 
+// ZigBee message
+message ZigBeeMsg {
 
-/* ZigBee messages */
+  // Zigbee command types definition
+  enum ZigBeeCmdType {
+    ZCL = 0;
+    ZLL = 1;
+    NETWORK_MGMT = 2;
+  }
 
-message ZigbeeBulbConfig {
-    // IP address or host name.
-    string address = 1;
-    // The port.
-    int32 port = 2;
-    // ZigBee command.
-    ZigBeeBulbCmd command = 3;
-}
+  // ZigBee Cluster Library command definition
+  message ZCLCmd {
+
+    // ZCL commands type
+    enum ZCLCmdType  {
+      ON_OFF = 0;
+      LEVEL = 1;
+      COLOR_CONTROL = 2;
+      IDENTIFY = 3;
+    }
+
+    // OnOff command definition
+    message OnOffCmd {
+
+      enum ZCLOnOffCmdType  {
+        ON = 0;
+        OFF = 1;
+        TOGGLE = 2;
+      }
+
+      // ZCL OnOff command type
+      ZCLOnOffCmdType type = 1;
+
+      // Params not required for ZCL OnOff cluster commands
+    }
+
+    // Level command definition
+    message LevelCmd {
+
+      // ZCL Level commands types
+      enum ZCLLevelCmdType  {
+        MOVE_TO_LEVEL = 0;
+        MOVE = 1;
+        // STEP = 2;
+        // STOP = 3;
+        // MOVE_TO_LEVEL_ONOFF = 4;
+        // MOVE_ONOFF = 5;
+        // STEP_ONOFF = 6;
+        // STOP_ONOFF = 7;
+      }
+
+       // MoveToLevel command parameters
+      message MoveToLevelCmdParams{
+        // level param <uint8>: The meaning of ‘level’ is device dependent – e.g. for a light it may mean brightness level.
+        uint32  level  = 1;
+        // Transition time <uint16>
+        uint32 transition_time  = 2;
+      }
+
+      // Move command parameters
+      message MoveCmdParams{
+        // |Fade Mode Value | Description  |
+        // |      0x00      |     Up       |
+        // |      0x01      |    Down      |
+        uint32  mode  = 1;
+        //Rate <uint8>: specifies the rate of movement in units per second.
+        uint32 rate  = 2;
+      }
+
+      ZCLLevelCmdType type = 1;
+
+      // MoveToLevel command
+      MoveToLevelCmdParams move_to_level_params = 2;
+
+      // Move command
+      MoveCmdParams move_params = 3;
+    }
+
+    // Color Control command definition
+    message ColorControlCmd {
+
+      // ZCL ColorControl command types
+      enum ZCLColorControlCmdType  {
+        MOVETOHUE = 0;
+        MOVETOSAT = 1;
+        MOVETOHUEANDSAT = 2;
+        // MOVEHUE = 3;
```

Comment:
Just add a note here. Something like:

// We do not yet support these commands

or

// We have no plans support these commands

or ...